lib/driver-cli.js

```diff
@@ -0,0 +1,91 @@
+import { KNOWN_DRIVERS, validateDriversJson } from './drivers';
+import { logger } from 'appium-support';
+const log = logger.getLogger('Driver CLI');
+
+let INSTALLED_DRIVERS;
+
+async function runDriverCommand (args) {
+  let jsonResult = null;
+  try {
+    INSTALLED_DRIVERS = validateDriversJson();
+    switch (args.driverCommand) {
+      case 'list':
+        jsonResult = await list(args);
+        break;
+      case 'install':
+        jsonResult = await install(args);
+        break;
+      case 'uninstall':
+        jsonResult = await uninstall(args);
+        break;
+      case 'update':
+        jsonResult = await update(args);
+        break;
+      default:
+        throw new Error(`Cannot handle driver command ${args.driverCommand}`);
```

Comment:
I'd rather show the list of supported commands

Reply:
Actually this is just a programming error. I wanted to include a default block merely out of completeness; argparse will fail long before we get here if someone tries to use a different command.

```
~/C/a/appium ❯❯❯ node . driver foo                                                                                                                       
usage: /Users/jlipps/Code/appium/appium driver [-h]
                                               {list,install,uninstall,update}
                                               ...
/Users/jlipps/Code/appium/appium driver: error: argument "driverCommand": Invalid choice: foo (choose from [list, install, uninstall, update])
```